docs/TheBook/src/main/markdown/cookbook-transport-security.md

```diff
@@ -8,7 +8,9 @@ To make sure data is safely encrypted during transfer, one should:
 1. Use the WebDAV protocol
 2. Configure dCache WebDAV doors not to redirect transfers (because currently it redirects to HTTP and not HTTPS)
 3. Have a well configured certificate or chain in each WebDAV door
-4. Have a well configured Java 8 or better
+4. Use Java 8 or better, with a well configured java.security
+
+Since 4.2, there is a new option `webdav.redirect.allow-https` which is `false` by default. If set to true, WebDAV doors should redirect clients to HTTPS movers on the pools. Your pools must have host certificates and have `pool.enable.encrypted-transfers = true` for this to work. This new feature should be considered experimental; test it before using it.
```

Comment:
As a writing style, we try to avoid phrases like "since 4.2" and "new option".

The reason is that each version of the book targets and describes a specific version of dCache.  For example, `master` branch targets (currently) dCache v5.1.  So it shouldn't mention dCache v5.0, v4.2, ...

Descriptions of how dCache has changed between  specific versions is (of course) very useful, especially for admins who are upgrading.   However, such version-comparing information belongs in the release notes, rather than in The Book.

As a suggestion, the following provides (roughly) the same information, but doesn't mention `4.2` or that the property is new:

    There is the `webdav.redirect.allow-https` option, which is `false` by default.  If set to true, ...

although you might like to rephrase the paragraph slightly, as the initial sentence is a little awkward in this version.